quality-control/human-clinical-and-health-data.md

```diff
@@ -1,27 +1,79 @@
 
-This is a community-driven website, so contributions are welcome! You will, of course, be listed as a contributor on the page.
+### Existing approaches
+* Clinical data collection: When collecting samples and data for an infectious disease, especially in the case of an outbreak, there are usually some established collection points. In these points, the identity of each tested individual is usually verified using an identification document (e.g., their ID). This allows the registration of their data in a platform used for data collection purposes. As one of the first quality assurance measures, it is important to double check the ID registered in the platform, one approach is comparing it with an existing database. In addition to the personal ID, a unique identifier should be created for each test performed, making it possible to distinguish among them, even if they are related to the same person ID.
 
-New content is announced on the [home page](/) and [news page](/about/news), so please check for updates there. You can also watch for changes on this page by using a free service like [Visual Ping](https://visualping.io/) or [Distill Web Monitor](https://distill.io/), or by using a [browser add-on](https://chrome.google.com/webstore/detail/distill-web-monitor/inlikjemeeknofckkjolnjbpehgadgge?hl=en).
+  Taking the example from the Czech Republic during the COVID-19 pandemic, the quality of COVID-19 connected clinical data is controlled by several systems. Firstly, all persons tested for the presence of SARS-CoV-2 virus by antigen or PCR tests were controlled on the collection points. The employees of the collection point controlled the IDs of the tested people and registered them to the Information system of infectious diseases (ISIN) of the Czech Republic. The correctness of the personal identification number was controlled against the population register of the Czech Republic during the registration of the person in the ISIN system. The unique personal identification number was generated for foreigners without a valid Czech personal identification number. Moreover, a unique ISIN request number was generated for all registered samples. This number served as a unique identifier of the test.
 
+  To ensure quality assurance of the clinical as well as biomolecular data and proper labelling and sample identification, CovIT software was developed by the Institute of Molecular and Translational Medicine (IMTM), Palacky University in Olomouc, Czech Republic. CovIT system has several modules. First module is designed for collection points. It controls the personal identification number, address, format of the phone number and communicates with the ISIN register.  
```

Comment:
It controls the personal identification number, address, format of the phone number and communicates with the ISIN register.   -> It controls the personal identification number, contact address, correct format of the phone number of the tested person and communicates with the ISIN register.